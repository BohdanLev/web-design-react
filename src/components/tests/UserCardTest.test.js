import React from 'react'; 
import UserCard from '../user-card';
import { shallow } from 'enzyme';


describe('UserCardComponent', () => {
    

      describe('UserCard container initial', () => {
        const UserCardContainer = (<UserCard/>)

        it('renders properly', () => {
            expect(UserCardContainer).toMatchSnapshot()
          })    
        
          const component = shallow(<UserCard/>)

        it('should render avatar', ()=>{
            expect(component.find('#user-avatar')).toBeDefined()
        })

        it('should render owner link and info', ()=>{
            expect(component.find('#link-to-owner')).toBeDefined()
        })
        
     })

     

})