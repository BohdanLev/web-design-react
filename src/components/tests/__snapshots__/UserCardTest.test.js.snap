// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCardComponent UserCard container initial renders properly 1`] = `<UserCard />`;
